airbyte-integrations/connectors/source-smartsheets/source_smartsheets/spec.json

```diff
@@ -17,6 +17,13 @@
         "title": "Sheet ID",
         "description": "The spreadsheet ID. Find in the spreadsheet menu: File > Properties",
         "type": "string"
+      },
+      "start_datetime": {
+        "title": "Start Datetime",
+        "type": "string",
+        "examples": ["2000-01-01T13:00:00", "2000-01-01T13:00:00-07:00"],
+        "description": "ISO 8601, for instance: `YYYY-MM-DDTHH:MM:SS`, `YYYY-MM-DDTHH:MM:SS+HH:MM`",
```

Comment:
I think we need add information about default value or set it in spec.

Reply:
Added default value to spec